infrastructure/http/YamlHTTPRequestParser.go

```diff
@@ -13,20 +13,29 @@ type YamlHTTPRequestParser struct{}
 //Parse ...
 func (instance YamlHTTPRequestParser) Parse(input map[string]interface{}) core.Action {
```

Comment:
Looking at the coverage this is covered so the tests didn't need to change.  The tests are higher level coverage though.